RevenueCatUI/CustomerCenter/Views/PurchaseHistory/PurchaseHistoryView.swift

```diff
@@ -28,91 +31,114 @@ struct PurchaseHistoryView: View {
-        return formatter
+    @ViewBuilder
+    private var contentView: some View {
+        ScrollViewWithOSBackground {
+            LazyVStack(spacing: 0) {
+                if viewModel.isLoading {
+                    ProgressView()
+                } else if viewModel.errorMessage != nil {
+                    ErrorView()
+                } else if !viewModel.isEmpty {
+                    if !viewModel.activeSubscriptions.isEmpty {
+                        PurchasesInformationSection(
+                            title: localization[.subscriptionsSectionTitle],
+                            items: viewModel.activeSubscriptions,
+                            localization: localization) { purchase in
+                                viewModel.selectedPurchase = purchase
+                            }
+                            .tint(colorScheme == .dark ? .white : .black)
+                    }
+
+                    if !viewModel.inactiveSubscriptions.isEmpty {
+                        PurchasesInformationSection(
+                            title: localization[.inactive],
+                            items: viewModel.inactiveSubscriptions,
+                            localization: localization) { purchase in
+                                viewModel.selectedPurchase = purchase
+                            }
+                            .tint(colorScheme == .dark ? .white : .black)
+                    }
+
+                    if !viewModel.nonSubscriptions.isEmpty {
+                        PurchasesInformationSection(
+                            title: localization[.purchasesSectionTitle],
+                            items: viewModel.nonSubscriptions,
+                            localization: localization) { purchase in
+                                viewModel.selectedPurchase = purchase
+                            }
+                            .tint(colorScheme == .dark ? .white : .black)
+                    }
+                }
+            }
+        }
     }
 }
 
 #if DEBUG
 @available(iOS 15.0, *)
 struct PurchaseHistoryView_Previews: PreviewProvider {
     static var previews: some View {
+        let customerInfo = CustomerInfoFixtures.customerInfo(
+            subscriptions: [
+                CustomerInfoFixtures.Subscription(
+                    id: "id1",
+                    store: "\(Store.appStore.rawValue)",
+                    purchaseDate: "2022-03-08T17:42:58Z",
+                    expirationDate: nil
+                )
+            ],
+            entitlements: [],
+            nonSubscriptions: [
+                CustomerInfoFixtures.NonSubscriptionTransaction(
+                    id: "id2",
+                    store: "\(Store.playStore.rawValue)",
+                    purchaseDate: "2022-03-08T17:42:58Z"
+                )
+            ])
+
         CompatibilityNavigationStack {
-            PurchaseHistoryView(viewModel: PurchaseHistoryViewModel(purchasesProvider: CustomerCenterPurchases()))
```

Comment:
it looks like `CustomerCenterPurchases` was removed. Maybe you can make it return the customerInfo above and that way you don't need to disable loading on previews?